### PR TITLE
Meeting Logs Kovri Dev (2018-06-07)

### PR DESCRIPTION
**Pull request by selsta**
https://gist.github.com/selsta/1082f90ae3fc7ce3af414e93ba37287a

### CODE REVIEW

**Comment by el00ruobuob** [comment]
Looks ok. I'll test it in the morning.

File: _posts/2018-06-07-logs-for-the-kovri-dev-meeting-held-on-2018-06-07.md [hunk 1]
```diff
@@ -0,0 +1,333 @@
+---
+layout: post
+title: Overview and Logs for the Kovri Dev Meeting Held on 2018-06-07
+summary: Discussion of the latest Kovri development updates and the alpha release
+tags: [dev diaries, i2p, crypto]
```
Comment (anonimal) [inline _posts/2018-06-07-logs-for-the-kovri-dev-meeting-held-on-2018-06-07.md]: Shouldn't we use `kovri` tag too? Is `kovri` used elsewhere?

Comment (el00ruobuob) [reply _posts/2018-06-07-logs-for-the-kovri-dev-meeting-held-on-2018-06-07.md]: According to other kovri log post, no : `tags: [dev diaries, i2p, crypto]`
i2p is the kovri tag on blog posts.

File: _posts/2018-06-07-logs-for-the-kovri-dev-meeting-held-on-2018-06-07.md [hunk 2]
```diff
@@ -0,0 +1,333 @@
+---
+layout: post
+title: Overview and Logs for the Kovri Dev Meeting Held on 2018-06-07
```
Comment (el00ruobuob) [inline _posts/2018-06-07-logs-for-the-kovri-dev-meeting-held-on-2018-06-07.md]: Other Kovri logs are like:
`title: Logs for the Kovri Dev Meeting Held on 2016-06-05`
This "Overview" is only used for monero dev meeting actually.
Whether i don't really care, at least, the file name should be relevant. As your filename is `2018-06-07-logs-for-the-kovri-dev-meeting-held-on-2018-06-07.md` i think the tilte should be `title: Logs for the Kovri Dev Meeting Held on 2018-06-07`

Comment (el00ruobuob) [reply _posts/2018-06-07-logs-for-the-kovri-dev-meeting-held-on-2018-06-07.md]: humm... obviously it has derived over time. Lots of blog title are not in line with their file name. Let's forget about my comment then.

File: _posts/2018-06-07-logs-for-the-kovri-dev-meeting-held-on-2018-06-07.md (hunk 1 above)
Comment (el00ruobuob) [reply _posts/2018-06-07-logs-for-the-kovri-dev-meeting-held-on-2018-06-07.md]: image
image

**Pull request merged**